client/systray/systray.go

```diff
@@ -160,6 +160,15 @@ func (menu *Menu) onReady() {
 	log.Printf("starting")
 	setAppIcon(disconnected)
 	menu.rebuild()
+
+	if menu.readonly {
```

Comment:
I think this needs to be behind `menu.mu.Lock`

Reply:
done